charts/config-server/templates/_overides.tpl

```diff
@@ -359,6 +359,12 @@ All env variables that are accessed from mosip config properties
       name: {{ .Values.overrides.configmaps.email }}
       key: email-smtp-port
 
+- name: SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_SMS_SMTP_PORT
+  valueFrom:
```

Comment:
SMS_PORT ..

Reply:
yes.. for email we are using 8025 for sms 8080 so i have updated like that ..